docker/test-base/Dockerfile

```diff
@@ -184,4 +184,6 @@ RUN git clone --depth=1 https://github.com/MacDownApp/macdown
 RUN git clone --depth=1 https://github.com/fossas/ant-example-project
 
 # Raw projects:
-RUN git clone --depth=1 https://github.com/jelaas/bifrost-framework
\ No newline at end of file
+RUN git clone --depth=1 https://github.com/jelaas/bifrost-framework
+
+RUN go get -u github.com/derekparker/delve/cmd/dlv
```

Comment:
This should really be in the base Dockerfile, which will cause all the Docker images to need a rebuild (groan).

Reply:
we can move this to the debug pr.